ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -579,6 +639,7 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
       // I think this is a callback function
       const videoNode = containerRef.current && containerRef.current.querySelector('video, audio');
+
```

Comment:
Some stuff a few lines below can be deleted. you caught the import of the aniview plugin above.